ndbench-cass-plugins/src/main/java/com/netflix/ndbench/plugin/cass/CJavaDriverBasePlugin.java

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2017 Netflix, Inc.  All rights reserved.
+ */
+package com.netflix.ndbench.plugin.cass;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.ConsistencyLevel;
+import com.datastax.driver.core.PreparedStatement;
+import com.datastax.driver.core.Session;
+import com.google.inject.Inject;
+import com.netflix.archaius.api.PropertyFactory;
+import com.netflix.ndbench.api.plugin.DataGenerator;
+import com.netflix.ndbench.api.plugin.NdBenchClient;
+import com.netflix.ndbench.api.plugin.common.NdBenchConstants;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author vchella
+ */
+public abstract class CJavaDriverBasePlugin implements NdBenchClient {
+
+    private static final org.slf4j.Logger Logger = LoggerFactory.getLogger(CJavaDriverBasePlugin.class);
+
+    protected static final String ResultOK = "Ok";
+    protected static final String ResutlFailed = "Failed";
```

Comment:
Since this is base class, I included all possible fields that get used in future extensions of this base class, hence kept it this way. Again, for same readability of the code.